starlite/config/logging.py

```diff
@@ -61,6 +215,35 @@ def configure(self) -> None:
                 break
         config.dictConfig(self.dict(exclude_none=True))
```

Comment:
I think we need to exclude the additional columns added that don't apply to the logger `dictConfig` (`middleware_*`)

Reply:
Should be better know.